config/options

```diff
@@ -52,6 +52,11 @@ LINUX_DEPENDS="$PROJECT_DIR/$PROJECT/linux/linux.$TARGET_ARCH.conf $ROOT/package
 # If you have ccache installed, take care that LOCAL_CC don't point to it
   LOCAL_CC=`which gcc`
 
+if [ -z "$LOCAL_CC" ] ; then
+  echo "***** y u no install gcc? *****"
```

Comment:
At the risk of sounding like I don't have a sense of humour, I'm not sure this message is entirely appropriate...